### PR TITLE
Add handling for new operation types to Redux state.

### PR DESCRIPTION
**Pull request by rahulgi**

### CODE REVIEW

File: src/store/reducers/members.ts [hunk 1]
```diff
@@ -251,6 +312,62 @@ function applyOperation(
...
+        const newMember = new Member({
+          ...memberData
+        });
+
+        return addMemberToState(prevState, newMember);
+      }
+      case OperationType.REQUEST_VERIFICATION: {
+        const { to_uid } = operation.data;
+
+        assertMemberIdPresentInState(prevState, creator_uid, operation);
```
Comment (markulrich) [inline src/store/reducers/members.ts]: Would it make sense to move all these `assertMemberIdPresentInState` into operationIsRelevantAndValid? Just at the beginning have something like if op in SET_CONST do the two asxertions?

File: src/store/reducers/members.ts [hunk 2]
```diff
@@ -31,28 +34,45 @@ const GENESIS_TRUST_OPS = [
...
-function getDefaultMemberFields() {
+function getDefaultMemberFields(): OptionalMemberFields {
   return {
     balance: new Big(0),
     totalDonated: new Big(0),
     totalMinted: new Big(0),
     trustedBy: Set<MemberId>(),
     invited: Set<MemberId>(),
-    trusts: Set<MemberId>()
+    invitedBy: undefined,
```
Comment (osdiab) [inline src/store/reducers/members.ts]: If you want to keep the lack of repetition, you can express this as  `undefined as MemberId | typeof GENESIS_MEMBER | undefined` and keep the old code

File: src/store/reducers/members.ts (hunk 1 above)
Comment (osdiab) [reply src/store/reducers/members.ts]: So long as the ids are predictable that would be fine

File: src/store/reducers/members.ts (hunk 2 above)
Comment (rahulgi) [reply src/store/reducers/members.ts]: TBH I like the explicit typedef. I find the above a little obtuse to parse lol

File: src/store/reducers/members.ts (hunk 1 above)
Comment (rahulgi) [reply src/store/reducers/members.ts]: I'll leave a TODO in for this for now - don't feel like modifying all the ops for this PR.

**Pull request merged**